Sources/NIOCore/ChannelOption.swift

```diff
@@ -12,6 +12,10 @@
 //
 //===----------------------------------------------------------------------===//
 
+#if os(Linux)
+import CNIOLinux
```

Comment:
Is this import needed?

Reply:
Nope, I previously had the is-supported check was on `DatagramGenericSegmentationOffload` but remember we have `System` which seemed a better fit.